server/routes/utils/reportHelper.ts

```diff
@@ -70,26 +73,30 @@ export const generatePDF = async (
       headless: true,
     });
     const page = await browser.newPage();
-    await page.goto(url, { waitUntil: 'networkidle0' });
 
+    await page.setDefaultNavigationTimeout(0);
     await page.setViewport({
       width: windowWidth,
       height: windowHeight,
+      deviceScaleFactor: 1,
```

Comment:
What's this option for

Reply:
Toggles the window size, but safe to remove, will do so